src/lib/colourType.js

```diff
@@ -12,4 +13,4 @@ export const isHex = colour => {
 
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
-export const isKeyword = colour => has(colourNames, colour.toLowerCase());
+export const isKeyword = colour => has.call(colourNames, colour.toLowerCase());
```

Comment:
This makes it not robust against `Function.prototype.call` being broken - but in all other respects, this would work fine. The goal of the "has" module is so you don't have to think about it nor code for it in all the many places in your app/dependency tree where you need this logic.